osu.Game/Overlays/Settings/SettingsFooter.cs

```diff
@@ -65,25 +62,49 @@ private void load(OsuGameBase game, OsuColour colours, RulesetStore rulesets)
                     Text = game.Name,
                     Font = OsuFont.GetFont(size: 18, weight: FontWeight.Bold),
                 },
-                new OsuSpriteText
+                new BuildDisplay(game.Version, DebugUtils.IsDebug)
```

Comment:
I guess it should check if the current build is deployed instead (as only deployed builds have a changelog page)

Reply:
i don't think this really matters at the end of the day.